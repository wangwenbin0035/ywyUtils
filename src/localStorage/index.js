/*
 * @Author: wwb 
 * @Date: 2020-04-16 18:22:14 
 * @Last Modified by: wwb
 * @Last Modified time: 2020-04-16 18:27:05
 */

export default {
  getStroage: (name) => {
    if (!name) return;
    return window.localStorage.getItem(name);
  },

  setStorage: (name,content) => {
    if (!name) return;
    if (typeof content !== 'string') {
      content = JSON.stringify(content);
    }
    window.localStorage.setItem(name, content);
  },

  deleteStroage: (name) => {
    if (!name) return;
    window.localStorage.removeItem(name);
  }
}