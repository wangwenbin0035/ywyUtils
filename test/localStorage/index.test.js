import localStorage from '../../src/localStorage';

describe('localStorage', () => {
  test('localStorage is MatchSnapshot', () => {
    expect(localStorage).toMatchSnapshot();
  });
});