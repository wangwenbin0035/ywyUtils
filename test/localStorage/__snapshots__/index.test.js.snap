// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`localStorage localStorage is MatchSnapshot 1`] = `
Object {
  "deleteStroage": [Function],
  "getStroage": [Function],
  "setStorage": [Function],
}
`;
